c2corg_images/resizing.py

```diff
@@ -46,12 +48,14 @@ def create_resized_images(path, key):
     if ext == '.svg':
         svg_key = key
         jpg_key = '{}{}'.format(base, '.jpg')
-        rasterize_svg(os.path.join(path, svg_key), os.path.join(path, jpg_key))
+        with stats.timer_context(['transform', 'rasterize_svg']):
+            rasterize_svg(os.path.join(path, svg_key), os.path.join(path, jpg_key))
         raster_file = os.path.join(path, jpg_key)
         key = jpg_key
 
     for config in RESIZING_CONFIG:
-        create_resized_image(path, key, config)
+        with stats.timer_context(['transform', 'resize']):
+            create_resized_image(path, key, config)
```

Comment:
You already measure this in create_resized_image.

Reply:
At least I did it with the same name ;-)